.gitignore

```diff
@@ -107,3 +107,8 @@ node_modules/
 #IDE specific ignores
 .idea/
 
+# Data files
+floodviz/static/data/gages.json
```

Comment:
Since we are using the data directory in a couple of places, I would suggest we ignore the whole directory. I believe Garrett did that in his pull request, so it will need to be resolved when merging.